finance.reckoner.app.yaml

```diff
@@ -0,0 +1,34 @@
+app-id: finance.reckoner.app
+runtime: org.gnome.Platform
+runtime-version: "44"
```

Comment:
current runtime is 45

Reply:
do you really need the GNOME runtime?

Reply:
Yes, the freedesktop platform did not interact with the keyring. The keyring is used to store database encryption password. If there is another way to save to keyring, I am open to testing it to see if it will work.

Reply:
this runtime will be EOL in less than 3 month. '45' is the current one.